test/unit/ping-centre/PingCentre.test.js

```diff
@@ -45,8 +45,8 @@ describe("PingCentre", () => {
 
     sandbox.stub(global.Services.prefs, "getBranch")
         .returns(new FakePrefs({initHook: prefInitHook}));
-    sandbox.stub(global.Services.locale, "getAppLocalesAsLangTags")
-        .returns([FAKE_LOCALE]);
+    sandbox.stub(global.Services.locale, "appLocaleAsLangTag")
+        .get(() => FAKE_LOCALE);
```

Comment:
Ha, I guess we only export activity stream stuff to newtab directory. Maybe we should just export this test as part of newtab especially once the tests are actually run from m-c.

Reply:
It would appear as though we're ignoring them on purpose? https://github.com/mozilla/activity-stream/blob/master/.mcignore#L17-L18